internal/backends/java/java.go

```diff
@@ -303,7 +309,9 @@ var JavaBackend = api.LanguageBackend{
 	Info:   info,
 	Add:    addPackages,
 	Remove: removePackages,
-	Install: func() {
+	Install: func(ctx context.Context) {
+		span, _ := tracer.StartSpanFromContext(ctx, "Maven install")
```

Comment:
```suggestion
		span, ctx := tracer.StartSpanFromContext(ctx, "Maven install")
```

Reply:
I wonder if we could avoid making a bunch of spans if we move these span definitions up one level in the call hierarchy and add the backend to the context.